### DISCUSSION TITLE
Account lost following change of smartphone & laptop

### DISCUSSION THREAD

**Discussion by RedouaneBALI** [community/community#131933] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

I recently realized that I lost access to my main account @Redouane59 because I used "Authenticator" app in a phone I recently changed. Unfortunately I also change my laptop at the same time and don't have recovery codes anymore.
The only thing I can confirm is my email address or my phone number.

It looks like **my GitHub account is definitively lost**, am I right ?

![image](https://github.com/user-attachments/assets/3e928d2f-3b03-4ceb-9f24-4f383a1e918c)

**Comment by Aditijainnn** [comment]
> ### Select Topic Area
> 
> Question
> ### Body
> 
> Hello,
> 
> I recently realized that I lost access to my main account @redouane59 because I used "Authenticator" app in a phone I recently changed. Unfortunately I also change my laptop at the same time and don't have recovery codes anymore. The only thing I can confirm is my email address or my phone number.
> 
> It looks like **my GitHub account is definitively lost**, am I right ?

 You can contact GitHub Support. Explain your situation, including details like your username, registered email address, and any other information that can help verify your identity such as previous commits, repositories you own or contributions, etc.
This can help you recover your account.

**Comment by ebndev** [comment]
Hi @RedouaneBALI,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
